index.html

```diff
@@ -2766,134 +2729,118 @@ <h3>Consumption</h3>
-    "signatureValue": "o9r6LxgoGN8FoaeeUA6EdDcv12GvDzFEmCgjWzvpur2YSQyA8W2r0SSWUK+nH5tMqzaFLun6wwZ1Eot37amGDg==",
-    "type": "ed25519Signature2018"
-  },
-  "verificationMethod": [
-    {
-      "curve": "ed25519",
-      "expires": "2019-12-01T03:00:00Z",
-      "id": "bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i;example:key=idbafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i",
-      "publicKeyBase64": "qmz7tpLNKKKdl7cD7PbejDiBVp7ONpmZbfmc7cEK9mg=",
-      "type": "EdDsaPublicKey"
-    }
-  ],
-  "updated": "2020-05-01T03:00:00Z"
-}
+<pre class="example nohighlight" title="DID Document as DagCBOR with tag 42 used in Proof section">
+  a5                                      # map(5)
+  62                                   # text(2)
+     6964                              # "id"
+  78 40                                # text(64)
+     6469643a6578616d706c653a313244334b6f6f574d4864727a6377706a6264725a7335474771455241766367715833623564707550745061396f743639796577 # "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew"
+  65                                   # text(5)
+     70726f6f66                        # "proof"
+  d8 2a                                # tag(42)
+     58 26                             # bytes(38)
+        000190011b20a7f3a13d0dafa9b8b640fee52173c27b9a66ff27f6669b3be73fc0ee6849e2ce # "\x00\x01\x90\x01\e \xA7\xF3\xA1=\r\xAF\xA9\xB8\xB6@\xFE\xE5!s\xC2{\x9Af\xFF'\xF6f\x9B;\xE7?\xC0\xEEhI\xE2\xCE"
+  68                                   # text(8)
+     40636f6e74657874                  # "@context"
+  78 1c                                # text(28)
+     68747470733a2f2f7777772e77332e6f72672f6e732f6469642f7631 # "https://www.w3.org/ns/did/v1"
+  6e                                   # text(14)
+     61757468656e7469636174696f6e      # "authentication"
+  81                                   # array(1)
+     78 83                             # text(131)
+        6469643a6578616d706c653a313244334b6f6f574d4864727a6377706a6264725a7335474771455241766367715833623564707550745061396f7436397965773b6b65792d69643d6261667972656963756274783577716f336e6f73633463617a726b63746668776436726577657a6770776f6534737769726c733465626468733269 # "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew;key-id=bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i"
+  72                                   # text(18)
+     766572696669636174696f6e4d6574686f64 # "verificationMethod"
+  81                                   # array(1)
+     78 40                             # text(64)
+        6469643a6578616d706c653a313244334b6f6f574d4864727a6377706a6264725a7335474771455241766367715833623564707550745061396f743639796577 # "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew"
           </pre>
         </section>
+      </section>
+    </section>
+    <section>
+      <h3>Converting Data between Core Representations</h3>
+      <p></p>
```

Comment:
```suggestion

      <div class="warning" title="Converting via DID document data model">
        <p>
This section conflicts with other parts of the specification, since
conversion has to be defined not between any two representations directly,
but rather in terms of parsing and serializing via the abstract
DID document data model (see <a href="#data-model"></a> and 
<a href="#representations"></a>). The content of this section can
potentially be adapted and re-used in other sections such as the one
about the CBOR representation (see <a href="#cbor"></a>).
        </p>
      </div>
```